### ISSUE TITLE
Incompatible changes

### ISSUE THREAD

**Issue by deg** [deg/sodium#4]
I am unhappy with some of the early names I chose in this library. `>evt` vs `>event` vs `event->fn` is awful to remember, and ditto for the sub and atom functions.

I'm in the process of moving these all into a new library, [Iron](https://github.com/deg/iron) so this is my best chance to introduce other breaking changes too.

I don't know if anyone else is using this library yet, nor if you will see this. If you are, and have doubts, this is your chance to contact me before I break anything for you.

**Comment by jamesnyika** [comment]
@deg I just came across the library and I was an avid user of re-com. I was going to use sodium but if you are moving everything to Iron.. I might wait. How soon do you plan on breaking stuff...

**Comment by deg** [comment]
Probably done in the next few days.

The changes will be in the names and namespaces of the support functions (like >event).

Are you interested in the Soda-ash half of Sodium or the utility functions  (or both)?
I don’t plan to change the former, while the latter is in the process of moving to Iron.

The rationale for Iron is that I’d like to eventually wrap baking-soda too (the library parallel to Soda-ash for Bootstrap).
I’ve already committed the first round of these changes but, as I cleaned up, I realized that some of my code had gotten ugly (like >evt and >event co-existing!)

David

**Comment by jamesnyika** [comment]
David,
 Thank you for your message. I am happy to wait for Iron if it will be a consistent version of what you intended Sodium to be. I really like what sodium offered (reduces my workload pulling data in and out of components). But I am looking for relative completeness and I was reading your code to see how exactly you pass params to soda-ash components. I think you did a fantastic job - the only things i saw in sodium that i needed to polyfill would have been stuff like the Card component and some of the others you had not gotten around to. 
 In short: I am interested in as much of soda-ash as you are willing to wrap and having the option to choose between semantic-ui components vs bootstrap would be fabulous! 

 Just go for it man.. ! 

James

**Comment by deg** [comment]
I've completed this round of changes. My test-app, [Trilystro](https://github.com/deg/trilystro) seems to have survived the changes, (and is also a decent source for examples using these libraries).

I think this is ready for use. Please let me know if you hit any snags.

Re continued enhancements and, especially, Bootstrap support: I hope to get to this. But, this is a part-time project and probably won't go much faster than my immediate needs for it. If you'd like to add features, PRs are very welcome, especially additional components and type checking.

**Comment by jamesnyika** [comment]
David
 I will certainly try out Iron as it exists now. I have not done much work with macros so it might take me a minute to understand what exactly you do to add a new component. Once I do that I might be able to add new components and send you some PRs. Thanks for this ..and access to your test app so I can see them at work... 

 Cheers!
